ui/src/app/layout/components/header/header.component.ts

```diff
@@ -75,4 +79,34 @@ export class HeaderComponent implements OnInit {
     selectEnvironment(env: KafkaEnvironment) {
         this.environments.setCurrentEnvironment(env);
     }
+
+    onDarkMode() {
+        if(localStorage.getItem('darkmode') === 'true') {
+            document.documentElement.classList.remove('dark');
+            document.getElementsByClassName('sidebar')[0].classList.remove('dark');
+            document.getElementsByClassName('toggle-button')[0].classList.remove('dark');
+            localStorage.setItem('darkmode', 'false');
+            this.darkMode = false;
+        }else {
+            document.documentElement.classList.add('dark');
+            document.getElementsByClassName('sidebar')[0].classList.add('dark');
+            document.getElementsByClassName('toggle-button')[0].classList.add('dark');
+            localStorage.setItem('darkmode', 'true');
+            this.darkMode = true;
+        }
+    }
+
+    initDarkMode() {
+        if(localStorage.getItem('darkmode') === 'true') {
+            document.documentElement.classList.add('dark');
+            document.getElementsByClassName('sidebar')[0].classList.add('dark');
+            document.getElementsByClassName('toggle-button')[0].classList.add('dark');
+            return true;
+        }else {
+            document.documentElement.classList.remove('dark');
+            document.getElementsByClassName('sidebar')[0].classList.remove('dark');
+            document.getElementsByClassName('toggle-button')[0].classList.remove('dark');
+            return false;
```

Comment:
Much code duplication between both methods. Could you check if simplification is possible?